docs/user_manual/type_data.md

```diff
@@ -715,6 +715,28 @@ is only deletable if it is not being used by any contacts.
 
 
 
+## Assay Tests
+
+[Assays](../requisitions#assays) detail the work intended to be performed for a requisition. This work is broken down
+into one or more tests. A test describes a workflow of extraction, library preparation, library qualification, and
```

Comment:
"workflow of X, Y, and Z" out of curiosity, are tests able to be created with only a subset of these workflow steps? (e.g. a test where we'd expect to receive libraries and perform only library qualification and full-depth sequencing) If so, perhaps `and/or` at the end here?

Reply:
Not really - the test describes all of these steps, whether they were performed internally or externally before receipt